src/tests/ftest/util/performance_test_base.py

```diff
@@ -396,8 +396,12 @@ def run_performance_ior(self, namespace=None, use_intercept=True, stop_delay_wri
 
             self.log.info("Running IOR read (%s)", str(iteration))
             self.ior_cmd.flags.update(read_flags)
-            self.ior_cmd.sw_wearout.update(None)
-            self.ior_cmd.sw_deadline.update(None)
+            if use_stonewalling_read:
+                self.ior_cmd.sw_wearout.update(write_sw_wearout)
+                self.ior_cmd.sw_deadline.update(write_sw_deadline)
+            else:
+                self.ior_cmd.sw_wearout.update(None)
+                self.ior_cmd.sw_deadline.update(None)
```

Comment:
This might not work how you want it to. IOR Write phase and Read phase are ran separately. Write phase writes to the `sw_status_file: "/var/tmp/daos_testing/stoneWallingStatusFile"` and Read phase reads from the same file so it knows how much to read. If you set a stonewall for Read phase as well, it might read past the end of the file if read is faster than write. I'd have to verify the exact behavior...